React/src/pages/orderStages/OrderStageCard.tsx

```diff
@@ -0,0 +1,454 @@
+                        <DatePicker
+                            sx={{
+                                width: '100%',
+                                '& .MuiInputBase-input.Mui-disabled': {
+                                    WebkitTextFillColor: '#000000',
+                                },
+                            }}
+                            label="Planowana data rozpoczęcia"
+                            disabled={isDisplayingMode}
+                            format="DD/MM/YYYY"
+                            value={plannedStartDate}
+                            onChange={(data) => {
+                                setError(null)
+                                setPlannedStartDate(data)
+                            }}
+                            onError={(error) => setError(error)}
+                            slotProps={{
+                                textField: {
+                                    helperText: errorMessage || '',
+                                },
+                            }}
+                            minDate={dayjs('2022-01-01T00:00:00.000')}
+                            maxDate={dayjs('2023-06-01T00:00:00.000')}
+                        />
+                    </LocalizationProvider>
+                </Grid>
+
+                {isDisplayingMode ? (
+                    <Grid item xs={6}>
+                        <LocalizationProvider dateAdapter={AdapterDayjs}>
+                            <DatePicker
+                                sx={{
+                                    width: '100%',
+                                    '& .MuiInputBase-input.Mui-disabled': {
+                                        WebkitTextFillColor: '#000000',
+                                    },
+                                }}
+                                label="Data rozpoczęcia"
+                                value={isDisplayingMode ? stage!.startDate : ''}
+                                disabled={isDisplayingMode}
+                            />
+                        </LocalizationProvider>
+                    </Grid>
+                ) : null}
+                <Grid item xs={6}>
+                    <LocalizationProvider dateAdapter={AdapterDayjs}>
+                        <TimePicker
+                            sx={{
+                                width: '100%',
+                                '& .MuiInputBase-input.Mui-disabled': {
+                                    WebkitTextFillColor: '#000000',
+                                },
+                            }}
+                            label="Planowana godzina rozpoczęcia"
+                            format={'HH:mm:ss'}
+                            value={plannedStartHour}
+                            disabled={isDisplayingMode}
+                            onChange={(data) => {
+                                setPlannedStartHour(data)
+                            }}
+                        />
```

Comment:
```suggestion
                        <TimePicker
                            sx={{
                                width: '100%',
                                '& .MuiInputBase-input.Mui-disabled': {
                                    WebkitTextFillColor: '#000000',
                                },
                            }}
                            label="Planowana godzina rozpoczęcia"
                            format={'HH:mm:ss'}
                            ampm={false}
                            value={plannedStartHour}
                            disabled={isDisplayingMode}
                            onChange={(data) => {
                                setPlannedStartHour(data)
                            }}
                        />
```
Zmieni to format godziny z 12H na 24H